model/common/tests/metric_tests/test_metric_fields.py

```diff
@@ -283,7 +280,7 @@ def test_compute_d2dexdz2_fac_mc(icon_grid, metrics_savepoint, grid_savepoint, b
         del_t_bg=del_t_bg,
         h_scal_bg=h_scal_bg,
         igradp_method=3,
-        igradp_constant=HorizontalPressureDiscretizationType.TAYLOR_HYDRO,
+        igradp_constant=3,
```

Comment:
Just a question: why inlining the constant value here instead of the previous approach?

Reply:
Because it the enum is defined in dycore. I don't know why `tach` did not capture this. Because it is in `tests` and we only check the `src` folders?

Reply:
Yes, that looks like the most likely reason.